azkaban-common/src/test/java/azkaban/executor/InteractiveTestJob.java

```diff
@@ -137,6 +138,12 @@ public void succeedJob(final Props generatedProperties) {
     }
   }
 
+  public void ignoreCancel() {
+    synchronized (this) {
+      this.ignoreCancel = true;
```

Comment:
again i think just making this boolean volatile or atomicboolean is sufficient, what do you think?